repository/o.json

```diff
@@ -771,6 +771,17 @@
 				}
 			]
 		},
+		{
+			"name": "Oracle Commerce Cloud Design Code Utility Putter",
+			"details": "https://github.com/byk0t/sublime-occ-putter",
+			"labels": ["occ", "oracle commerce cloude", "design code utility"],
```

Comment:
These labels can probably be removed since it is unlikely there will be multiple plugins with them